layers/+tools/shadowenv/packages.el

```diff
@@ -0,0 +1,40 @@
+;;; packages.el --- Shadowenv Layer packages File for Spacemacs
+;;
+;; Copyright (c) 2012-2021 Sylvain Benner & Contributors
+;;
+;; Author: Sylvain Benner <sylvain.benner@gmail.com>
+;; URL: https://github.com/syl20bnr/spacemacs
+;;
+;; This file is not part of GNU Emacs.
+;;
+;; This program is free software; you can redistribute it and/or modify
+;; it under the terms of the GNU General Public License as published by
+;; the Free Software Foundation, either version 3 of the License, or
+;; (at your option) any later version.
+;;
+;; This program is distributed in the hope that it will be useful,
+;; but WITHOUT ANY WARRANTY; without even the implied warranty of
+;; MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+;; GNU General Public License for more details.
+;;
+;; You should have received a copy of the GNU General Public License
+;; along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+
+(defconst shadowenv-packages
+  '(shadowenv))
+
+(defun shadowenv/init-shadowenv ()
+  (use-package shadowenv
+    :defer t
```

Comment:
optionally we can use
`:spacediminish " Ⓥ" " V"`

It takes 0 to 3 arguments, same as those of `spacemacs|diminish`.
When the first argument is omitted, it's inferred from the package name, which is `(intern (format "%s-mode" PKG-NAME))`.

Reply:
Thanks for mentioning `:spacediminish`, I was not aware of it.

Done.